### DISCUSSION TITLE
How to resync failed files.

### DISCUSSION THREAD

**Discussion by XuRong233** [minio/minio#15422] [Q&A] [answered]
How to resync failed files.
mc replicate status minio-server/image
Summary:                                                                                       
Replication Status    | Size (Bytes)    | Count          
   Failed             | 66 MiB          | 4,967          
   Replicated         | 9.2 GiB         |                
   Replica            | 1.4 GiB         |                

Remote Target Statuses:                                                                        
● arn:minio:replication::3bbbfb82-f2f5-429b-9e9d-a4c5bfc51917:image                                          
Replication Status    | Size (Bytes)    | Count          
   Failed             | 0 B             | 0              
   Replicated         | 8.5 MiB         |                

● arn:minio:replication::f1c22776-4242-45c4-aed9-1c024044ac18:image                                          
Replication Status    | Size (Bytes)    | Count          
   Failed             | 66 MiB          | 4,967          
   Replicated         | 9.2 GiB         |        
## Your Environment

* Version used (`2022-07-17T15:43:14Z`):
* Server setup and configuration:Two sites, each site uses two servers and 4 drivers.
* Operating System and version (`Linux localhost.localdomain 3.10.0-1160.el7.x86_64 #1 SMP Mon Oct 19 16:18:59 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`):

**Comment by poornas** [comment, marked as answer]
The replication failures will heal automatically when the inbuilt healing process kicks in, this depends on the scanner speed in revisiting this part of namespace on your cluster. You can tweak the scanner speed with `mc admin config set minio-server scanner delay=1s max_wait=1s cycle=10s` , this may impact your cluster performance if scanner is too aggressive and you may have to monitor and dial it back to default.

In 2022-07-24 minio release and latest mc there is a `mc replicate diff minio-server/image` command that can help identify the unreplicated files. If there is only a handful, these can be forced to replicate manually by setting a tag on the object version in question with `mc tag set minio-server/image/<obj-name> --vid <version-id> "k=v"`. This metadata update will in turn force replication to pick it up right away. The tag can be removed afterwards

**Comment by XuRong233** [comment]
Thank you, the copy failure has been automatically repaired
